source/train/DescrptSeA.py

```diff
@@ -1,13 +1,14 @@
 import numpy as np
 from deepmd.env import tf
-from deepmd.common import ClassArg, get_activation_func, get_precision
+from deepmd.common import ClassArg, get_activation_func, get_precision, argproperty
 from deepmd.RunOptions import global_tf_float_precision
 from deepmd.RunOptions import global_np_float_precision
 from deepmd.env import op_module
 from deepmd.env import default_tf_session_config
 
 class DescrptSeA ():
-    def __init__ (self, jdata):
+    @argproperty
+    def args(cls):
         args = ClassArg()\
                .add('sel',      list,   must = True) \
```

Comment:
```suggestion
               .add('sel',      list,   must = True, docs="the maximum possible number of neighbors in the cut-off radius.") \
```
@amcadmus @jameswind The detailed document needs to be filled like this.

Reply:
After all things are done, I will consider generating a web page.